themes/default/content/docs/guides/saml/aad.md

```diff
@@ -98,7 +98,9 @@ download the SAML application's configuration data and then pass that to Pulumi.
     ![Download XML](/images/docs/reference/service/saml-aad/download-xml.png)
 
 1. Sign into the Pulumi Service and navigate to your SAML organization. Navigate to the
-  **Settings** tab and then select **SAML SSO**.
+  **Settings** tab and then select **Access Management**.
+  
+1. Click the **Change Requirements** button and select **SAML SSO**.
```

Comment:
@MitchellGerdisch we try and avoid the use of the word `click` in our docs because it is pointer centric. this and a few other things are documented in our style guide: https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md

```suggestion
1. Select the **Change Requirements** button and then **SAML SSO**.
```